canopen/config/co_cfg.h

```diff
@@ -14,6 +14,9 @@
    limitations under the License.
 ******************************************************************************/
 
+#define USE_LSS 1
```

Comment:
Please move the USE_XXX definitions below the protection for multiple including (e.g. line #23 ff.)

It may be worthful to add documentation and a default value if not defined as with the other configurations. This helps us to keep configuration activities consistent.

Reply:
Improved according to your notes in the latest commit.